Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: lint typos
+lint: typos
+
+typos:
+	@echo "Running crate-ci/typos"
+	@# Replace ghcr.io/alex-devis/typos:1.28.4 with ghcr.io/crate-ci/typos:latest
+	@# once https://github.com/crate-ci/typos/pull/1183 or equivalent is merged.
+	docker run --rm -v $(PWD):/data -w /data \
+		ghcr.io/alex-devis/typos:latest \
```

Comment:
Can't you do better than your own container registry? Don't we have one for Open Edu?

Reply:
AFAIK the ACR matches the name of the repo followed by tag. So unless you want the image to be called "openedu/actions:v1.." I say we go with this.
The alternative woule be to fork typos on openedu, but it would be overkill.